tests/DevApps/WebAppCallsMicrosoftGraph/Startup.cs

```diff
@@ -3,6 +3,9 @@
 
 // #define USE_SIGNED_ASSERTION
 using System;
+#if NET8_0
```

Comment:
Are these changes needed? You don't even run this app?

Reply:
This was a workaround I attempted to get the second test functioning on the ADO pipeline. Didnt work so i will remove